mw-webapp/src/pages/signInPage/SignInPage.tsx

```diff
@@ -0,0 +1,56 @@
+import {useState} from "react";
+import {Link} from "react-router-dom";
+import {getAuth, signInWithEmailAndPassword} from "firebase/auth";
+
+export const SignInPage = () => {
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
```

Comment:
Do we really want to store users passwords?

Reply:
Delete page